### DISCUSSION TITLE
Ruby update only when there is a new latest version

### DISCUSSION THREAD

**Discussion by tyl3r-ch** [postmodern/ruby-install#440] [Q&A]
Hi ruby-install experts,
How is it possible with ruby-install to install the latest version only in case it is not already installed?

```sh
ruby-install --update ruby
```

At the moment the version 3.2.0 is installed, when I run this again, the same version is installed again. Since this takes some time, it would be helpful if an installation is only done if the version is not already present.
Best Regards

**Comment by postmodern** [comment]
I'm just curious why you are running `ruby-install --update ruby` when there is no new ruby version announced? You should only run `ruby-install --update ruby` when a new version is released. Unless `ruby-install --update ruby` is being called from another bash script?

Currently, `ruby-install` will re-install the ruby, even if it was previously previously installed into the installation directory. There is a `--no-reinstall` option that will abort the installation if `bin/ruby` is detected within the installation directory.
